integration/spark-common-test/src/test/scala/org/apache/carbondata/spark/testsuite/dataretention/DataRetentionConcurrencyTestCase.scala

```diff
@@ -18,44 +18,63 @@
 package org.apache.carbondata.spark.testsuite.dataretention
 
 import java.util
-import java.util.concurrent.{Callable, Executors, Future}
+import java.util.concurrent.{Callable, ExecutorService, Executors, Future}
 
 import scala.collection.JavaConverters._
 
 import org.apache.spark.sql.test.util.QueryTest
-import org.scalatest.BeforeAndAfterAll
+import org.scalatest.{BeforeAndAfterAll, BeforeAndAfterEach}
 
 import org.apache.carbondata.core.constants.CarbonCommonConstants
+import org.apache.carbondata.core.datamap.dev.DataMapWriter
+import org.apache.carbondata.core.datamap.dev.cgdatamap.{CoarseGrainDataMap, CoarseGrainDataMapFactory}
+import org.apache.carbondata.core.datamap.{DataMapDistributable, DataMapMeta, Segment}
+import org.apache.carbondata.core.datastore.page.ColumnPage
+import org.apache.carbondata.core.metadata.AbsoluteTableIdentifier
+import org.apache.carbondata.core.metadata.schema.table.DataMapSchema
+import org.apache.carbondata.core.scan.filter.intf.ExpressionType
 import org.apache.carbondata.core.util.CarbonProperties
+import org.apache.carbondata.events.Event
 
 
 /**
  * This class contains DataRetention concurrency test cases
  */
-class DataRetentionConcurrencyTestCase extends QueryTest with BeforeAndAfterAll {
+class DataRetentionConcurrencyTestCase extends QueryTest with BeforeAndAfterAll with BeforeAndAfterEach {
```

Comment:
I think this testcase is not valid anymore, please delete this suite

Reply:
@jackylk already deleted in current master hence i am closing this pr